get_images.sh

```diff
@@ -7,7 +7,8 @@ if [ ! -f "$RHCOS_IMAGE_FILENAME" ]; then
     curl --insecure --compressed -L -o "${RHCOS_IMAGE_FILENAME}" "${RHCOS_IMAGE_URL}/${RHCOS_IMAGE_VERSION}/${RHCOS_IMAGE_FILENAME}".gz
 fi
 
-mkdir -p "$IRONIC_DATA_DIR/html/images"
+sudo mkdir -p "$IRONIC_DATA_DIR/html/images"
+sudo chown $USER "$IRONIC_DATA_DIR/html/images"
```

Comment:
Maybe change ownership for all the files within directory?

```suggestion
sudo chown -R $USER "$IRONIC_DATA_DIR/html/images"
```